internal/core/services/scheduler_manager/scheduler_manager.go

```diff
@@ -88,6 +88,10 @@ func (s *SchedulerManager) GetScheduler(ctx context.Context, schedulerName, vers
 	})
 }
 
+func (s *SchedulerManager) GetSchedulerVersions(ctx context.Context, schedulerName string) ([]*entities.SchedulerVersion, error) {
```

Comment:
@caiordjesus i think the test for this new guy is missing

Reply:
Actually the test for GetScheduler was missing too!
Just added on the last commit. Can you please take a look again?